managing-os/templates.md

```diff
@@ -87,6 +87,17 @@ Important Notes
 
        $ sudo yum remove qubes-template-fedora-24
 
+ * Standalone VMs using Template VMs as a basis can be created easily. These
+   VMs receive a *copy* of the operating system and do not get automatically
+   updated when Template VMs are updated--they must be updated individually.
+   
+ * Template VMs can occupy more space on the dom0 filesystem than necessary
+   because they cannot employ automatic TRIM/discard on the root fs. The
+   `qvm-trim-template` command in dom0 is used to recover this unused space.
```

Comment:
Well... I seem [unable to reproduce](https://gist.github.com/anonymous/19416b4227f32d426c2bc52172aae156) my [results from a StandaloneVM](https://github.com/QubesOS/qubes-issues/issues/2479#issuecomment-264674619) in a TemplateVM's root filesystem mounted with discard enabled, so I must conclude that the statement is indeed correct.

Sorry for wasting your time.